tests/test_clientsecrets.py

```diff
@@ -222,12 +224,18 @@ def test_validate_error(self):
             except clientsecrets.InvalidClientSecretsError as e:
                 self.assertTrue(str(e).startswith(match))
 
-    def test_load_by_filename(self):
+    def test_load_by_filename_missing_file(self):
+        exc_caught = None
         try:
             clientsecrets._loadfile(NONEXISTENT_FILE)
-            self.fail('should fail to load a missing client_secrets file.')
-        except clientsecrets.InvalidClientSecretsError as e:
-            self.assertTrue(str(e).startswith('File'))
+        except clientsecrets.InvalidClientSecretsError as exc:
+            exc_caught = exc
+
+        expected_args = ('Error opening file',
```

Comment:
:+1: For some reason that has been the style of `oauth2client` tests.

There isn't a single test in `gcloud-python` that checks **ANY** of the content of an exception.